app/policies/activity_policy.rb

```diff
@@ -14,6 +14,14 @@ def create?
     record.organisation == user.organisation
   end
 
+  def create_child?
+    return record.fund? if beis_user?
+    return false if record.third_party_project?
+    return false unless editable_report?
```

Comment:
Why does there need to be an editable report in existence in order for a new activity to be created?

Reply:
Well, it's matching the [existing behaviour](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/1041/files/a5e852050964c43ef778fa9f2a32bd81d69f7faa..56710ca14a2bc3c506ee8abf40b9d00f44decca5#diff-d9caded5b030bd11eaf9e5847ec15f8fbc14f2f357a0aa27da212846c6a0a736L13) in the `create?` method. I will check however!

Reply:
Did we find an answer to this question?

Reply:
We're going to take this out for now, although there is a user need to prevent anything being created or edited whilst reports are being approved. This will be added after some more thought has been put into it.